### PR TITLE
Merge IdToken changes to develop

### PR DESCRIPTION
**Pull request by swetha-gadiraju**
Changes include 
- Added new API getIdToken() in Flutter plugin
- Android & iOS implementations for the new API
- Added new wrapper error 'idTokenNotAvailable' when id token value is null from native SDK
- Unit tests for Android
- Config model updates for Android & iOS

### CODE REVIEW

File: example/ios/Configuration/OneginiConfigModel.m [hunk 1]
```diff
@@ -6,14 +6,14 @@ @implementation OneginiConfigModel
...
 }
 
 + (NSDictionary *)configuration
 {
     return @{
-             @"ONGServerType" : @"Access",
-             @"ONGServerVersion" : @"0809c19",
+             @"ONGServerType" : @"access",
```
Comment (szymon-onewelcome) [inline example/ios/Configuration/OneginiConfigModel.m:15]: Was this done manually or by the configurator? I think it should be still with a capital letter.

Comment (swetha-gadiraju) [reply example/ios/Configuration/OneginiConfigModel.m:15]: It was by configurator and works with small letters "access". I can see the same small letter in iOS example app as well.

File: android/build.gradle [hunk 2]
```diff
@@ -65,7 +65,7 @@ android {
 }
 
 dependencies {
-    api('com.onegini.mobile.sdk.android:onegini-sdk:12.0.0@aar') {
+    api('com.onegini.mobile.sdk.android:onegini-sdk:12.1.1@aar') {
```
Comment (aleknow) [inline android/build.gradle:68]: in 12.1.0 we changed the type of the error returned when registation is aborted by developer https://developer.onewelcome.com/android/sdk/v12-X#improvements. Did you check if any changes in the code are needed? (I guess nothing has to be changed) We should also mention that in the release notes for new flutter version

**Review by aleknow — APPROVED** [review]

File: android/build.gradle (hunk 2 above)
Comment (swetha-gadiraju) [reply android/build.gradle:68]: No additional code changes required. The new error message "The registration action was canceled by the user" is passed back through Flutter plugin as well.

**Review by szymon-onewelcome — APPROVED** [review]

**Pull request merged**